src/walletConnect/v2/saga.ts

```diff
@@ -51,6 +51,8 @@ let client: SignClient | null = null
 
 const TAG = 'WalletConnect/saga'
 
+const CONNECTION_TIMEOUT = 10_000
```

Comment:
Can we use a different name so there's no confusion with the the other `CONNECTION_TIMEOUT`?

Maybe?
```suggestion
const GET_SESSION_TIMEOUT = 10_000
```

Reply:
yikes i thought i pushed a change to update this but it didn't go through, i'll address this in the next PR